src/test/java/com/hillel/items_exchange/util/CategoryTestUtil.java

```diff
@@ -0,0 +1,114 @@
+package com.hillel.items_exchange.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import com.hillel.items_exchange.dto.CategoryDto;
+import com.hillel.items_exchange.dto.SubcategoryDto;
+import com.hillel.items_exchange.model.Category;
+import com.hillel.items_exchange.model.Subcategory;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class CategoryTestUtil {
```

Comment:
I'm going to refactor current class...

Reply:
Maybe next time because it`s not connected with functionality and I`m afraid the QA team doesn`t have enough time for testing this huge ticket.